docs/install.rst

```diff
@@ -54,4 +54,52 @@ so that you can do changes and submit patches.
 
 You can also find tox packaged for many Linux distributions and Homebrew for macOs - usually under the name of **python-tox** or simply **tox**. Be aware though that there also other projects under the same name (most prominently a `secure chat client <https://tox.chat/>`_ with no affiliation to this project), so make sure you install the correct package.
 
+Installation of all needed Python versions
+------------------------------------------
+
+As you install tox for different projects, you may notice that it wants to run tests with different versions of Python,
+some of which are not installed on your system.
+This section aims to help you further by providing an overview over different guides that help installing and using
+different versions of Python next to each other on one operating system.
```

Comment:
We need to have a disclaimer here that we do **not** offer support to install Python.